### PR TITLE
Add documentation to root/socialnet/includes/sn_core_addons.php

### PR DESCRIPTION
**Pull request by senky**
Adding phpDocumentor2 comments before each function, class and method in root/socialnet/sn_core_addons.php

### CODE REVIEW

File: root/socialnet/includes/sn_core_addons.php [hunk 1]
```diff
@@ -1,25 +1,53 @@
...
+ */
 if (!defined('SOCIALNET_INSTALLED') || !defined('IN_PHPBB'))
 {
 	return;
 }
 
+/**
+ * Core Addons class
+ *
+ * @package phpBB-Social-Network
```
Comment (Culprit-CZ) [inline root/socialnet/includes/sn_core_addons.php:27]: Creating another package. One is defined on line 10
I think that should be the same names

Comment (senky) [inline root/socialnet/includes/sn_core_addons.php:27]: Ok, can we use `phpBB-Social-Network`? I think using dashes (`-`) gives us much more advantages (in exporting data by way that does not allow spaces, etc.).

Comment (Culprit-CZ) [inline root/socialnet/includes/sn_core_addons.php:27]: Without dashes looks more better

**Comment by Kamahl19** [comment]
Is this already finished?

**Comment by Culprit-CZ** [comment]
No is not finished, but as first documentation for this file.
MA

**Pull request merged**